ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -145,17 +149,25 @@ export default defineComponent({
         if (localState.feeType === FeeSummaryTypes.RENEW) {
           feeSummary.processingFee = 5
         }
+        if (props.setFeeOverride) {
+          feeSummary.feeAmount = props.setFeeOverride.feeAmount
+        }
+        if (props.setFeeOverride && feeSummary.serviceFee !== 0) {
+          feeSummary.serviceFee = props.setFeeOverride.serviceFee
+        }
         return feeSummary
       }),
       hasProcessingFee: computed(() => {
         return props.setStaffReg || props.setStaffSBC
       }),
       hintFee: computed((): string => {
-        return getFeeHint(
+        const hint = getFeeHint(
           localState.feeType,
           localState.registrationType,
           localState.registrationLength
         )
+        if (props.setFeeOverride) return hint.replace('$5', '$0')
```

Comment:
If the hint changes from $5?

Reply:
this is just changing the *$5.00 / year* to *$0.00 / year*. The *$5.00 / year* never changes regardless of how many years they select, except for when it's a different hint (i.e. infinite..) and in that case we don't need anything to change